packages/lms/src/lib/components/content/Main.svelte

```diff
@@ -12,5 +12,32 @@
 	main {
 		grid-area: body;
 		overflow-y: auto;
+
+		/* for firefox users*/
+		scrollbar-color: var(--primary) var(--background-primary);
+		scrollbar-width: thin;
+
+		max-width: 100ch;
+	}
+
+	::-webkit-scrollbar {
+		width: 6px;
+	}
+	::-webkit-scrollbar-track {
+		box-shadow: inset 0 0 4px var(--primary);
+		border-radius: var(--border-radius-xs);
+		background-color: var(--background-primary);
+	}
+	::-webkit-scrollbar-thumb {
+		background-color: var(--primary);
+		border-radius: var(--border-radius-xs);
+	}
+
+	@media screen and (max-width: 935px) {
+		main {
+			overflow: inherit;
+			grid-area: 3 / 1 / 4 / 5;
+			padding-bottom: 3rem;
+		}
```

Comment:
I remember seeing something in the docs/project tasks about preferring container queries over media queries. Did we step away from this?

Reply:
Nope, I would like to enforce @container queries throughout unless @media is utterly unavoidable

Reply:
This works fine for me, but think the main content should have some margin left and right as it's kind of hugging the edges as the screen gets smaller

Reply:
I've added some padding now